src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -157,6 +159,24 @@
     )
     private String note;
 
+    @CCD(
+        label = "Digital Notice of Proceedings?",
+        access = {CaseworkerAndSuperUserAccess.class}
+    )
+    private YesOrNo digitalNoticeOfProceedings;
+
+    @CCD(
+        label = "Notice of Proceedings email address",
+        typeOverride = Email,
+        access = {CaseworkerAndSuperUserAccess.class}
+    )
+    private String noticeOfProceedingsEmail;
+
+    @CCD(
+        label = "Notice of Proceedings solicitor's firm"
+    )
+    private String noticeOfProceedingsSolicitorFirm;
```

Comment:
Are these fields only for sole applications when issuing an AOS? If so, I wonder if they should live in the AcknowledgmentOfService class

Reply:
I moved thesse fields into AcknowledgmentOfService
https://github.com/hmcts/nfdiv-case-api/pull/429/commits/516defb3338cc6b7f7155a938c7388e1b4417e79